conans/client/rest/uploader_downloader.py

```diff
@@ -12,10 +12,11 @@ def __init__(self, requester, output, verify, chunk_size=1000):
         self.requester = requester
         self.verify = verify
 
-    def post(self, url, content):
+    def post(self, url, content, auth=None):
```

Comment:
I know it was already like this, but I would like to rename the "post" method to upload. It's weird a post method doing a put